tests/python/gaia-ui-tests/gaiatest/apps/homescreen/regions/confirm_dialog.py

```diff
@@ -22,3 +22,7 @@ def tap_confirm(self, bookmark=False):
             *self._confirm_button_locator))
         Wait(self.marionette).until(expected.element_displayed(confirm))
         confirm.tap()
+        
+        from gaiatest.apps.homescreen.regions.bottom_bar import BottomBar
+        return BottomBar(self.marionette)
```

Comment:
I think it's better to just use BottomBar(self.marionette).tap_done() in test_homescreen_delete_app.py instead of returning this object from here.